src/main/webapp/resources/js/components/OutputFiles/OutputFileHeader.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Component to render a file header (filename and download button)
+ * for each file on preview page
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Button, Icon, Col, Typography } from "antd";
+import { convertFileSize } from "../../utilities/file-utilities";
+import { downloadOutputFile } from "../../apis/analysis/analysis";
+import { SPACE_MD, SPACE_XS } from "../../styles/spacing";
+import { FONT_SIZE_DEFAULT } from "../../styles/fonts";
+import styled from "styled-components";
+
+const { Text } = Typography;
+
+const OutputFileHeaderWrapper = styled.div`
+  margin-bottom: ${SPACE_MD};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+/**
+ * Stateless UI component for creating the file header with download button
+ * @param {object} output - The current output file
+ *
+ * @returns {Element} - Returns a component which has a file header with download button
+ */
+
+export function OutputFileHeader({ output }) {
+  return (
+    <OutputFileHeaderWrapper>
+      <Col>
+        <Text
+          style={{
+            fontSize: FONT_SIZE_DEFAULT
+          }}
+        >
+          {`${output.toolName} ${output.toolVersion} - ${output.outputName} - ${output.filename}`}
+        </Text>
+      </Col>
+      <Col>
+        <Button
+          style={{
+            marginLeft: SPACE_XS
+          }}
+          onClick={() =>
+            downloadOutputFile({
+              submissionId: output.analysisSubmissionId,
+              fileId: output.id
+            })
+          }
+        >
```

Comment:
You should be able to just use the attribute `icon="download"` on the button instead of needing to add an icon component.

Reply:
Updated in e22c19a4382eef23663a043a8c05f36ea3f10cec